### DISCUSSION TITLE
Function Lock Enhancement

### DISCUSSION THREAD

**Discussion by nilanjan-mkj** [seerge/g-helper#1896] [Ideas]
Currently when the Function Lock feature is turned on, the Hotkeys function properly without clicking the Fn button. But when simultaneously holding the fn button and the function keys (suppose f4 in Excel) the keys do not revert back to normal function key behaviour.....they continue to perform as hotkeys even when fn button is pressed simultaneously. To bring back the function keys we need to turn off the function lock feature. Can this be made a bit more seamless.

**Comment by seerge** [comment]
They don't cause it's not hardware fn-lock, g-helper just imitates that by turning F1-F11 row into own hotkeys when lock is ON. Reverse is not doable :)
